### PR TITLE
Feature/mypy

### PR DESCRIPTION
**Pull request by Zethson**
please make it stop

### CODE REVIEW

File: cookietemple/common/suggest_similar_commands.py [hunk 1]
```diff
@@ -22,7 +23,7 @@ def load_available_handles() -> set:
     :return: A set of all available handles
     """
     available_templates = load_yaml_file(f'{AVAILABLE_TEMPLATES_PATH}')
-    unsplit_handles = set()
+    unsplit_handles: Set[str] = set()
     all_handles = set()
```
Comment (Imipenem) [inline cookietemple/common/suggest_similar_commands.py:27]: add type hint Set here

File: cookietemple/cookietemple_cli.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ def create(domain: str) -> None:
 
 
 @cookietemple_cli.command(short_help='Lint your existing cookietemple project.', cls=CustomHelpSubcommand)
-@click.argument('project_dir', type=click.Path(), default=Path(str(Path.cwd())), helpmsg='Relative path to projects directory.', cls=CustomArg)
+@click.argument('project_dir', type=click.Path(), default=Path(str(Path.cwd())), helpmsg='Relative path to projects directory.', cls=CustomArg)  # type: ignore
 @click.option('--skip-external', is_flag=True, help='Only run cookietemple linting and not external linters.')
```
Comment (Imipenem) [inline cookietemple/cookietemple_cli.py:98]: might need to refactor this here: type and default are looking weird, just default to `Path.cwd()`

File: cookietemple/cookietemple_cli.py [hunk 3]
```diff
@@ -218,8 +218,9 @@ def sync(project_dir, set_token, pat, username, check_update) -> None:
 
 
 @cookietemple_cli.command('bump-version', short_help='Bump the version of an existing cookietemple project.', cls=CustomHelpSubcommand)
-@click.argument('new_version', type=str, required=False, helpmsg='New project version in a valid format.', cls=CustomArg)
-@click.argument('project_dir', type=click.Path(), default=Path(f'{Path.cwd()}'), helpmsg='Relative path to the projects directory.', cls=CustomArg)
+@click.argument('new_version', type=str, required=False, helpmsg='New project version in a valid format.', cls=CustomArg)  # type: ignore
+@click.argument('project_dir', type=click.Path(), default=Path(f'{Path.cwd()}'),
```
Comment (Imipenem) [inline cookietemple/cookietemple_cli.py:222]: like above, we should default to `Path.cwd()`

File: cookietemple/cookietemple_cli.py (hunk 2 above)
Comment (Zethson) [reply cookietemple/cookietemple_cli.py:98]: Yeah. But note that this PR does not change anything here.

**Review by Imipenem — COMMENTED** [review]
What about adding the modified release WF and mypy WF to the other templates (mypy for sure only python projects)?

File: cookietemple/cookietemple_cli.py (hunk 2 above)
Comment (Imipenem) [reply cookietemple/cookietemple_cli.py:98]: Yes, I know, I just noticed it ;D

**Comment by Zethson** [comment]
> "What about adding the modified release WF and mypy WF to the other templates (mypy for sure only python projects)?

I'll address this in another PR.
We should merge this today to allow you to modify the code.

**Comment by Imipenem** [comment]
> I'll address this in another PR.
> We should merge this today to allow you to modify the code.

Fine with me, can merge

**Pull request merged**